tests/components/ozw/test_cover.py

```diff
@@ -84,3 +87,47 @@ async def test_cover(hass, cover_data, sent_messages, cover_msg):
     msg = sent_messages[4]
     assert msg["topic"] == "OpenZWave/1/command/setvalue/"
     assert msg["payload"] == {"Value": 0, "ValueIDKey": 625573905}
+
+
+async def test_barrier(hass, cover_gdo_data, sent_messages, cover_gdo_msg):
+    """Test setting up config entry."""
+    receive_message = await setup_ozw(hass, fixture=cover_gdo_data)
+    # Test loaded
+    state = hass.states.get("cover.gd00z_4_barrier_state")
+    assert state is not None
+    assert state.state == "closed"
+
+    # Test opening
+    await hass.services.async_call(
+        "cover",
+        "open_cover",
+        {"entity_id": "cover.gd00z_4_barrier_state"},
+        blocking=True,
+    )
+    assert len(sent_messages) == 1
+    msg = sent_messages[0]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 4, "ValueIDKey": 281475083239444}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5795327)